File: speedtest01.pl
#!/usr/bin/env perl
use strict;
use warnings;

use constant END_LOOP => 1_000_000_000; # 1 Mrd.

my $last_number = 0;

for (my $i = 0; $i < END_LOOP; $i++) {
    $last_number++;
}

print "Perl:last_number=$last_number\n--\n";

File: speedtest01.php
<?php
define('END_LOOP', 1000000000); // 1 Mrd.

function main() {
    $last_number = 0;
    for ($i = 0; $i < END_LOOP; $i++) {
        $last_number += 1;
    }

    echo "PHP:last_number=$last_number\n--";
}

main();
?>

